node/src/node_configurator/configurator.rs

```diff
@@ -2132,29 +2301,36 @@ mod tests {
     fn make_example_generate_wallets_request() -> UiGenerateWalletsRequest {
         UiGenerateWalletsRequest {
             db_password: "password".to_string(),
-            mnemonic_phrase_size: 24,
-            mnemonic_phrase_language: "English".to_string(),
-            mnemonic_passphrase_opt: Some("booga".to_string()),
-            consuming_derivation_path: derivation_path(0, 4),
-            earning_derivation_path: derivation_path(0, 5),
+            seed_spec_opt: Some(UiGenerateSeedSpec {
+                mnemonic_phrase_size: 24,
+                mnemonic_phrase_language: "English".to_string(),
+                mnemonic_passphrase_opt: Some("booga".to_string()),
+            }),
+            consuming_derivation_path_opt: Some(derivation_path(0, 4)),
+            earning_derivation_path_opt: Some(derivation_path(0, 5)),
         }
     }
 
-    fn make_example_recover_wallets_request() -> UiRecoverWalletsRequest {
+    fn make_example_recover_wallets_request_with_paths() -> UiRecoverWalletsRequest {
         UiRecoverWalletsRequest {
             db_password: "password".to_string(),
-            mnemonic_phrase: vec![
-                "parent", "prevent", "vehicle", "tooth", "crazy", "cruel", "update", "mango",
-                "female", "mad", "spread", "plunge", "tiny", "inch", "under", "engine", "enforce",
-                "film", "awesome", "plunge", "cloud", "spell", "empower", "pipe",
-            ]
-            .into_iter()
-            .map(|x| x.to_string())
-            .collect::<Vec<String>>(),
-            mnemonic_passphrase_opt: Some("ebullient".to_string()),
-            mnemonic_phrase_language: "English".to_string(),
-            consuming_derivation_path: derivation_path(0, 4),
-            earning_wallet: "0x005e288d713a5fb3d7c9cf1b43810a98688c7223".to_string(),
+            seed_spec_opt: Some(UiRecoverSeedSpec {
+                mnemonic_phrase: vec![
+                    "parent", "prevent", "vehicle", "tooth", "crazy", "cruel", "update", "mango",
+                    "female", "mad", "spread", "plunge", "tiny", "inch", "under", "engine",
+                    "enforce", "film", "awesome", "plunge", "cloud", "spell", "empower", "pipe",
```

Comment:
If you want you might use the "make_.." instant method to set up the phrase as well as you've done at other places.